pkg/sink/codec/canal/canal_json_row_event_encoder.go

```diff
@@ -348,7 +348,7 @@ func (c *JSONRowEventEncoder) AppendRowChangedEvent(
 			zap.Int("maxMessageBytes", c.maxMessageBytes),
 			zap.Int("length", length),
 			zap.Any("table", e.Table),
-			zap.Any("value", value))
```

Comment:
There  is already  a length value above.

Reply:
:+1:

Reply:
@nongfushanquan Even the PK might contain PII. I removed the duplicated length logging. Didn't realize it was already there.